tracex_project/tracex/logic/utils.py

```diff
@@ -1,26 +1,36 @@
 """Module providing various utility functions for the project."""
 import os
-from io import StringIO
-from pathlib import Path
+import json
 
 import base64
 import tempfile
 import functools
 import warnings
+
+from io import StringIO
+from pathlib import Path
+
 import pandas as pd
 import pm4py
 import numpy as np
+import requests
 
 from django.conf import settings
 from django.db.models import Q
 from openai import OpenAI
+
 from tracex.logic.logger import log_tokens_used
 from tracex.logic.constants import (
     MAX_TOKENS,
     TEMPERATURE_SUMMARIZING,
     MODEL,
     oaik,
 )
+from tracex.logic.constants import (
```

Comment:
Why add another import statement for the snomed stuff when we already import things like max_tokens from constants?

Reply:
true